soundbay/models.py

```diff
@@ -1,11 +1,17 @@
 import importlib
+from typing import Union
+
 import torch
+import torchaudio
 import torch.nn as nn
 from torch import Tensor
 from torchvision.models.resnet import ResNet, BasicBlock, conv3x3, Bottleneck
 from torchvision.models.vgg import VGG
 from torchvision.models import squeezenet, ResNet18_Weights
 import torchvision.models as models
+from torchaudio.models import wav2vec2_model
```

Comment:
This is not necassary if we `import torchaudio`

Reply:
yep you are right